src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ProvidePropertyAttribute.cs

```diff
@@ -24,7 +24,7 @@ public sealed class ProvidePropertyAttribute : Attribute
             }
 
             PropertyName = propertyName;
-            ReceiverTypeName = receiverType.AssemblyQualifiedName;
+            ReceiverTypeName = receiverType.AssemblyQualifiedName!; // TODO-NULLABLE: ReceiverTypeName should be guarded against nulls
```

Comment:
I'll stop commenting on these, but same goes for all of them (either address in this PR or open an issue).